Dockerfile

```diff
@@ -68,6 +78,14 @@ RUN mkdir -p ./src/bin && \
 # Build real binaries now
 COPY ./src ./src
 COPY ./components ./components
+COPY ./cmd ./cmd
+
+# Remove profiling feature
+RUN sed -i '/^profiling/d' ./cmd/Cargo.toml
+
+# Add cmd back
+RUN sed -i '/"components\/pd_client",/a\ \ "cmd",' Cargo.toml && \
+    sed -i '/X_CARGO_CONFIG_FILE=${DIST_CONFIG}/ax-build-dist: export X_PACKAGE=cmd' Makefile
```

Comment:
How about just using `COPY` to copy the infomation about cmd back here?

Reply:
If using `COPY` here, the following commands need to be executed again
```Dockerfile
# Remove fuzz and test workspace, remove profiler feature
RUN sed -i '/fuzz/d' Cargo.toml && \
    sed -i '/test\_/d' Cargo.toml && \
    sed -i '/profiler/d' Cargo.toml
```
I'm not sure which one is better.